keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -426,7 +440,7 @@ rules:
   resourceNames:
   - example.com/my-signer
```

Comment:
I believe this is implemented in admission so it also supports the `example.com/*` wildcard which should be mentioned here.

Reply:
Added a sentence below to point this out.